.github/scripts/generate_ascii_report.py

```
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)